### PR TITLE
added kernel_tuner and myself

### PR DESCRIPTION
**Pull request by benvanwerkhoven**

### CODE REVIEW

File: _software/kernel_tuner.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- Accelerated Computing
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/b.vanwerkhoven
+owner:
+- http://software.esciencecenter.nl/organization/nlesc
+- http://software.esciencecenter.nl/person/b.vanwerkhoven
+contributor:
+- http://software.esciencecenter.nl/person/b.vanwerkhoven
+user:
+involvedOrganization:
```
Comment (blootsvoets) [inline _software/kernel_tuner.md]: Any projects or organizations?

File: _software/kernel_tuner.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+discipline:
+- eScience Methodology
+expertise:
+- High Performance Computing
+- Accelerated Computing
+supportLevel: specialized
+contactPerson: http://software.esciencecenter.nl/person/b.vanwerkhoven
+owner:
+- http://software.esciencecenter.nl/organization/nlesc
+- http://software.esciencecenter.nl/person/b.vanwerkhoven
```
Comment (blootsvoets) [inline _software/kernel_tuner.md]: Any other organizations on a project that could actually be co-owner?

**Comment by benvanwerkhoven** [comment]
No other (co-)owners. No projects yet that have used it, but there will be many, I will be using it in the NFI project for example. Hanno said that he will probably use it in the neutrino project. This is very likely relevant for every project that develops GPU code.

**Comment by benvanwerkhoven** [comment]
So what happens now?

**Pull request merged**